aksel.nav.no/website/app/dev/_ui/Avatar.tsx

```diff
@@ -0,0 +1,101 @@
+import Image from "next/image";
+import { BodyShort, HStack } from "@navikt/ds-react";
+import { BLOGG_LANDINGSSIDE_BLOGS_QUERYResult } from "@/app/_sanity/query-types";
+import styles from "./Avatar.module.css";
+import { InteractiveAvatarStack } from "./InteractiveAvatarStack";
+
+const MAX_AVATAR_COUNT = 30;
+
+export const avatarUrl = (avatar_id: string) => {
+  let _avatar_id = avatar_id;
+  if (!isNaN(parseInt(avatar_id))) {
+    _avatar_id =
+      `${parseInt(avatar_id) % MAX_AVATAR_COUNT}`.padStart(3, "0") ?? "broken";
+  }
+  return `/avatars/${_avatar_id}.svg`;
+};
+
+type Blogg = NonNullable<BLOGG_LANDINGSSIDE_BLOGS_QUERYResult>["bloggposts"][0];
+
+export type Avatar = {
+  imageSrc: string;
+  name: string;
+  description: string;
+};
+
+export const queryToAvatars = (
+  queryResponseEditorialStaff: Blogg["editorial_staff_teams"],
+): Avatar[] => {
+  return (
+    queryResponseEditorialStaff?.map((queryData) => ({
+      name: queryData.title ?? "",
+      imageSrc: avatarUrl(queryData.avatar_id?.current ?? "missing"),
+      description: queryData.description ?? "",
+    })) ?? []
+  );
+};
+
+const AvatarCircle = ({ avatar }: { avatar: Avatar }) => {
+  return (
+    <Image
+      className={styles.avatarImage}
```

Comment:
Adding `priority` and maybe `loading="eager"` will make the initial loading more snappy